content/docs/user-guide/experiment-management/cleaning-experiments.md

```diff
@@ -63,6 +78,9 @@ branches in this example.
 
 ```dvc
 $ dvc exp gc --workspace
+WARNING: This will remove all experiments except those derived from the workspace of the current repo. Run queued experiments will be removed.
+Are you sure you want to proceed? [y/n]: y
+Removed 5 experiments. To remove unused cache files use 'dvc gc'.
 $ dvc exp list --all
```

Comment:
Not sure we need so much detail in the sample output, could just be an empty line (or `...`). Conveys the idea without noise

```suggestion
$ dvc exp gc --workspace

$ dvc exp show --all-tags
```